test/joint_tours/configs/joint_tour_frequency_composition.csv

```diff
@@ -0,0 +1,99 @@
+Label,Description,Expression,Coefficient
+,Constant for shopping tour,@df.shopping,0
+,Constant for Maintenance tour,@df.othmaint,-1.477
```

Comment:
I think all of these constants should be moved to the joint_tour_frequency_composition_coeffs.csv file which is currently empty.